.gitmodules

```diff
@@ -20,6 +20,6 @@
         ignore = all
 [submodule "sorc/MPASSIT"]
 	path = sorc/MPASSIT
-	url = https://github.com/rrfsx/MPASSIT
+	url = https://github.com/LarissaReames-NOAA/MPASSIT.git
```

Comment:
Larissa's version does not work on Gaea
The module file for Hera and Jet are not standard.
I think the best way is to sync what we need fro Larissa's version to the rrfsx fork until Larissa merges this PR https://github.com/LarissaReames-NOAA/MPASSIT/pull/42

Reply:
This is to match the version used in real-time MPAS runs on Jet. We will figure out how to manage MPASSIT repository after we can repeat realtime MPAS runs.

Reply:
@hu5970 We can make it to repeat what David has on Jet and at the same work for Hera/Gaea as well. 

@daviddowellNOAA @hu5970 Do we still use a precompiled mpassit.x on Jet?  If not, could you let me know what branch/hash you have been using for Jet realtime runs from Larrissa's repo? I think it is very possible to repeat Jet realtime runs without scarifying other machines. Thanks!

Reply:
For Jet real-time runs, we're using the most current version (hash number bd91387) of code in https://github.com/LarissaReames-NOAA/MPASSIT.

Reply:
Thanks, @daviddowellNOAA!

@hu5970 I've rebased RRFSx/MPASSIT up to Larissa's latest commit # bd91387
https://github.com/RRFSx/MPASSIT/commits/master/
Could you go ahead to point sorc/MPASSIT to the above latest master branch?

@daviddowellNOAA My commits did not involve any source code changes, only unifying the MPASSIT modulefiles across all platforms to use spack-stack.v1.6.0.  It can compile on Jet. You are welcome to try that.
If you want to do a quick test, here is the precompiled executable:
`/lfs5/BMC/wrfruc/gge/rrfs-workflow/exec/mpassit.x`
and be sure to load the following module before running MPASSIT:
```
module use sorc/MPASSIT/modulefiles
module load build.jet.intel
```